Adds class file for container test

hasukmistry opened the pull request.

Commit:
Adds class file for container

wesolowski committed:
Member: $key is private and can`t not by use in SodiumEncryption (#270)

wesolowski committed:
Member has private access and can`t not by use in: (#269)

- APC@saveItem
- APCU@saveItem
- FileCache@saveItem
- Memcache@saveItem
- Memcached@saveItem
- Redis@saveItem
- SessionCache@saveItem

wesolowski committed:
Bugfix: many variable names are not there (#268)

* Bugfix: many variable names are not there

* Fix: https://github.styleci.io/analyses/86WvBN?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

hasukmistry committed:
Adds convert case method in str class and adds tests (#267)

* Adds ConvertCase methods and test cases

* Adds missing parameter information in function documentation

* Formates the code to align the standard

* Aligns the code standard

* Returns value when mbstring is not active

peter279k commented on Tests/Container/ContainerTest.php:
```diff
@@ -0,0 +1,9 @@
+
+class ContainerTest extends TestCase
+{
+}
```
This Container test doesn't have any tests.

Please add at least one test to do `Container` class test.

lablnet replied:
Yes you're right

hasukmistry replied:
`use Zest\Container\Container` I added this statement first.  Because we did not have any test cases there, the usage of that class did not occur in it. So an automatic code check was complaining about it. Then later I removed that statement. I can add a test case to it. Can anyone suggest a test here? :) @peter279k @Lablnet

lablnet replied:
You may looks to others frameworks test in case of `Laravel` ....

hasukmistry replied:
@Lablnet  and @peter279k, I have implemented one test for container and also modified two functions in container. It will be great if you look at it. :)

lablnet reviewed (changes requested):
Add some `tests` this is nothing you do.

2mt-heuser committed:
Add isBase64 and substring methods to Str class (#272)

* Add isBase64 and substring methods to Str class

* Fix code-style issues

* Adjust code according to review

* Fix docblock to incorporate null as possible value for encoding

* Add testcase for optional parameter length & encoding

Commit:
Adds empty method inside test class

Commit:
Add String methods  (#279)

* Add Base64 to Zest\Data\Str #271

* Add Base64 to Zest\Data\Str #271

 - Add tests

* Add substring to Zest\Data\Str #271

 - Add function
 - Add tests

* Add substring to Zest\Data\Str #271

 - Add function
 - Add tests
 - PHPDoc

* Add stripWhitespaces to Zest\Data\Str #271

 - Add function
 - Add tests
 - PHPDoc

* Add repeat to Zest\Data\Str #271

 - Add function
 - Add tests
 - PHPDoc

Fixed
 - PHPDoc for stripWhitespaces

* Add slice to Zest\Data\Str #271

 - Add function
 - Add tests
 - PHPDoc

* Add shuffle to Zest\Data\Str #271

 - Add function
 - Add tests
 - PHPDoc

* StyleCI recommendations

* apply styleci

* Fix scrutinizer issues

* StyleCI

* Requested changes

* Requested change

Commit:
We do not need shell file anymore

It seems operating system depended

Commit:
Update changes.md

Commit:
Update CONTRIBUTORS.md

2mt-heuser committed:
Fix behaviour of hasUppercase and hasLowercase (#280)

* Fix behaviour of hasUppercase and hasLowercase

* Fix code-style issue

* Remove unnecessary whitespace

ReeceM committed:
feat: add the option to output assoc arrays fixes #261 (#281)

* feat: add the option to output assoc arrays /fix #261

* fix: apply styleci and missing contract definition

* styleci

* fix: apply styleci and missing contract definition

* fix: apply review suggestion.

Commit:
Update Configuration.php

lablnet committed:
Change notification email

lablnet committed:
Update PasswordManipulation.php

lablnet committed:
Apply fixes from StyleCI (#283)

lablnet committed:
Update PasswordManipulation.php

Commit:
Update .travis.yml (#285)

Commit:
Create PasswordManipulationTest.php (#284)

* Create PasswordManipulationTest.php

* Update PasswordManipulationTest.php

* fix typos

* Update PasswordManipulationTest.php

* Update PasswordManipulation.php (#287)

* Update Site.php (#288)

* Update Site.php

* Update Site.php

* Update PasswordManipulation.php

* Update Site.php

* Update PasswordManipulation.php

* Update helpers.php (#289)

* styling

* Update Site.php

* Update PasswordManipulation.php

* Update PasswordManipulation.php

* Update Site.php

* Update PasswordManipulation.php

* Update changes.md (#290)

Commit:
Added FileInfo Class

Commit:
Apply fixes from StyleCI (#291)

Commit:
Update FileInfo Class

Commit:
Added Key Class:

Commit:
Update KeyTest.php

Commit:
Fix syntax error

Commit:
Apply fixes from StyleCI (#292)

Commit:
Update Key.php

Commit:
Adds test for container and modifies class functions

Commit:
Removes extra spaces

Commit:
Merge branch 'feature/add-testclass-container' of https://github.com/hasukmistry/Zest_Framework into feature/add-testclass-container
